tests/common_tests/test_tp_model.py

```diff
@@ -25,7 +26,7 @@
 tp = mct.target_platform
 
 TEST_QC = generate_test_op_qc(**generate_test_attr_configs())
-TEST_QCO = tp.QuantizationConfigOptions([TEST_QC])
+TEST_QCO = model_compression_toolkit.target_platform_capabilities.schema.v1.QuantizationConfigOptions([TEST_QC])
```

Comment:
I think that at least in the tests you should definitely use an alias for this path.
Also, a reference to a "current schema" pointer is very important for the tests as well (we want the tests to run with the active schema used by MCT without needing to change all this imports each time we update a schema)